build.gradle

```diff
@@ -48,20 +48,14 @@ repositories {
 
 dependencies {
     compile gradleApi()
-    compile 'com.android.tools.build:gradle:1.5.0'
-    compile 'com.google.code.gson:gson:2.5'
```

Comment:
I'm confused by this; how does this even build without gson?  We explicitly use it in `PackageTree#printJson()`; I guess that there must be a transient dependency.  I'd prefer to keep the dependency explicit.